lib/connection/connection_config.js

```diff
@@ -834,6 +868,7 @@ const PARAM_RETRY_SF_MAX_LOGIN_RETRIES = 'sfRetryMaxLoginRetries';
 const PARAM_RETRY_SF_MAX_NUM_RETRIES = 'sfRetryMaxNumRetries';
 const PARAM_RETRY_SF_STARTING_SLEEP_TIME = 'sfRetryStartingSleepTime';
 const PARAM_RETRY_SF_MAX_SLEEP_TIME = 'sfRetryMaxSleepTime';
+const PARAM_QUERY_CONTEXT_CAHCE_SIZE = 'queryContextCacheSize';
```

Comment:
it's QUERY_CONTEXT_CACHE_SIZE

Reply:
If it's not being removed, please fix the typo "CAHCE"